src/services/DidService.ts

```diff
@@ -34,7 +34,7 @@ class DidService {
     } as IContact)
 
     const status = await did.store(myIdentity.identity)
-    if (status.isError) {
+    if (status.isEmpty) {
```

Comment:
With the changes, it seems that we cannot have the `isError` not sure what the equivalent is. 

Will check more about the types to find a better solution.